pkg/scheduler/framework/plugins/volumerestrictions/volume_restrictions.go

```diff
@@ -37,6 +43,10 @@ const Name = "VolumeRestrictions"
 const (
 	// ErrReasonDiskConflict is used for NoDiskConflict predicate error.
 	ErrReasonDiskConflict = "node(s) had no available disk"
+	// ErrReasonUnexpected is used when unexpected errors occur checking volume restrictions.
```

Comment:
what type of unexpected errors here?

Reply:
Not found errors are an example, in which case we should retry to allow time for the PVC to propagate to the lister.